src/Content/Backend/Solution/Monaco.Template.Backend.Application.Infrastructure.Migrations/Monaco.Template.Backend.Application.Infrastructure.Migrations.csproj

```diff
@@ -10,4 +10,8 @@
     <ProjectReference Include="..\Monaco.Template.Backend.Application.Infrastructure\Monaco.Template.Backend.Application.Infrastructure.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Migrations\" />
+  </ItemGroup>
```

Comment:
```suggestion

```

Reply:
Yeah, I didn't care to remove this one as it will be created by the EF Core CLI anyway when creating the first migration, so...